src/engraving/libmscore/beam.cpp

```diff
@@ -2249,7 +2249,7 @@ void Beam::editDrag(EditData& ed)
     // Because of the logic in Beam::setProperty(),
     // changing Pid::BEAM_POS only has an effect if Pid::USER_MODIFIED is true.
     undoChangeProperty(Pid::USER_MODIFIED, true);
-    undoChangeProperty(Pid::BEAM_POS, PointF(y1 / _spatium, y2 / _spatium));
+    undoChangeProperty(Pid::BEAM_POS, QVariant::fromValue(qMakePair(y1 / _spatium, y2 / _spatium)));
```

Comment:
this was preventing using the mouse from editing Beams (my fault, I'd changed this property to use a `QPair<float,float>` a while back but missed a few places)